lib/texutil.js

```diff
@@ -43,10 +45,19 @@ module.exports.latex_function_names = arr2set([
 ]);
 
 module.exports.paren = arr2set(["\\sinh"]);
-try{ //try catch to make sure program still runs without csv file
-    var csv = require('csv-parse/lib/sync');
-    var fs = require('fs');
-    var path = require('path');
+module.exports.other_literals3 = obj2map({});
+var csv = require('csv-parse/lib/sync');
+var fs = require('fs');
+var path = require('path');
+try {
+    module.exports.other_literals3 = parseSty(fs.readFileSync(path.join(__dirname, "texvc.sty"), 'utf8'));
+}
+catch (e) {
+    if (e.code !== "ENOENT") {
+        throw e;
+    }
```

Comment:
The file is in the github repo. So it it's OK to error out, the file was not found.